src/assets/icons/ArrowRight.tsx

```diff
@@ -0,0 +1,21 @@
+import { forwardRef } from 'react';
+
+import { Icon, IconProps } from '@/component-library/Icon';
+
+const ArrowRight = forwardRef<SVGSVGElement, IconProps>((props, ref) => (
+  <Icon
+    ref={ref}
+    xmlns='http://www.w3.org/2000/svg'
+    stroke='none'
+    viewBox='0 0 24 24'
+    fill='currentColor'
+    strokeWidth='0'
```

Comment:
Why is this set to "0"?

Reply:
Doesn't need to be—just needs to be removed. I've tidied that up.